modules/katamari/src/test/ts/atomic/api/Regex/RegexTest.ts

```diff
@@ -0,0 +1,16 @@
+import { Assert, UnitTest } from '@ephox/bedrock-client';
+import * as Regex from 'ephox/katamari/api/Regex';
+
+UnitTest.test('Regex.escapeRegExp', () => {
```

Comment:
Shouldn't this be mocha style and chai we recently switched to that.

Reply:
updated